tests/fps_tests.py

```diff
@@ -136,6 +136,34 @@ def test_inequal_cases(self):
         fps_2.service_sites = ["https://service2.com"]
         self.assertNotEqual(fps_1, fps_2)
 
+class TestFpsIncludes(unittest.TestCase):
+    def test_primary_case(self):
+        fps = FpsSet(ccTLDs={
+                        "https://primary.com": "https://primary.ca"
```

Comment:
```suggestion
                        "https://primary.com": ["https://primary.ca"]
```

The ccTLD dict entries should be lists, shouldn't they? That way there could be multiple different aliases for the same "canonical" domain. Same comment elsewhere.

(Side note - it might be a good idea to add some static type checking in this script, via mypy or something else. Separate PR, of course.)